src/Powercord/plugins/pc-spotify/commands/search.js

```diff
@@ -0,0 +1,38 @@
+const SpotifyAPI = require('../SpotifyAPI');
+
+module.exports = {
+    command: 'search',

```

Comment:
This command is already in use by [another plugin](https://github.com/Juby210/quick-search), which would cause a conflict

Reply:
Renamed to find